docs/how-to-build.md

```diff
@@ -151,3 +155,197 @@ This software is licensed under the Apache License version 2."
    This creates `gravitino-trino-connector-{version}.tar.gz` and
    `gravitino-trino-connector-{version}.tar.gz.sha256` under the `distribution` directory. You 
    can uncompress and deploy it to Trino to use the Gravitino Trino connector.
+
+## How to Build Gravitino on Windows (Using WSL)
+### Download WSL (Ubuntu)
+
+**On Windows:**
+
+Refer to this guide for installation: [WSL Installation Guide](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+*Note: Ubuntu 22.04 can successfully run Gravitino*
+
+
+This step involves setting up the Windows Subsystem for Linux (WSL) on your Windows machine. WSL allows you to run a Linux distribution alongside Windows, providing a Linux-like environment for development.
+
+### Update Package List and Install Necessary Packages
+
+**On Ubuntu (WSL):**
+
+```sh
+sudo apt update
+sudo apt install apt-transport-https ca-certificates curl software-properties-common
+```
+
+Updating the package list ensures you have the latest information on the newest versions of packages and their dependencies. Installing the necessary packages sets up your system to securely download and manage additional software.
+
+### Download and Setup Java SDK 17 (11 or 8 also works)
+
+**On Ubuntu (WSL):**
+
+1. Edit your `~/.bashrc` file using any editor. Here, `vim` is used:
+
+    ```sh
+    vim ~/.bashrc
+    ```
+
+2. Add the following lines at the end of the file. Replace `/usr/lib/jvm/java-11-openjdk-amd64` with your actual Java installation path:
+
+    ```sh
+    export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
+    export PATH=$PATH:$JAVA_HOME/bin
+    ```
+
+3. Save and quit in vim using `:wq`.
+
+Editing the `~/.bashrc` file allows you to set environment variables that will be available in every terminal session. Setting `JAVA_HOME` and updating `PATH` ensures that your system uses the correct Java version for development.
+
+### Install Docker
+
+**On Ubuntu (WSL):**
+
+Refer to chatGPT for specific instructions. Here are the steps:
```

Comment:
Have these steps been validated locally?

Reply:
yes, I made it to install docker on WSL.